rero_ils/modules/acq_orders/templates/rero_ils/vendor_order_mail/eng.tpl.txt

```diff
@@ -9,14 +9,24 @@ We would like to order the following list of books:
 
 ------------------------------------------------------------
 {%- for line in order.order_lines %}
-Title : {{ line.document.title }}
-{%- if line.document.identifiers %}
-ISBN : {{ line.document.identifiers | join(', ') }}
-{%- endif %}
+Document :
+  {{ line.document.title_text }}
+  {%- if line.document.provision_activity %}
+  Provision activity : {{ line.document.provision_activity }}
```

Comment:
In 99% of the cases, the provision activity is a publication activity, so it's clearer to put here Publication statement.

```suggestion
  Publication statement : {{ line.document.provision_activity }}
```

Reply:
OK for me